Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+namespace Lesson3
+{
+    public class Lesson3
+    {
+	    public Lesson3()
+	    {
+            static void Main(string[] args)
+            {
+                string name = PromptString("name");
```

Comment:
Please split to a function

Reply:
I will need additional clarification to understand this one, sorry. Do you mean you want lines 11-16 within a function and then you only want main to call that function? That seems redundant so maybe that is not what you mean.

Reply:
Yes, line 11-18 to be precise. In highest level functions, such as main, we try not to put the implementation of other functions. If we can make it a high level (function), we do. Ideally, you should have a quick look at it and understand what it does. right now I need to read 8 lines of code in order to understand what happens.

What happens? ``ProcessPersonBmi()`` or similar.

Reply:
I am still unclear on the rationale behind this and accordingly, I am not sure how to proceed. What I am hearing is "put all this code together into one equally confusing and arbitrary function then call that function." I will proceed with that but I feel that is not what is being asked of me.